addons/website/data/website_data.xml

```diff
@@ -176,8 +162,8 @@
                                         </div>
                                     </div>
                                     <div class="col-lg-4">
```

Comment:
The "thank you" page is strange to customize, because of the `me-lg-auto` class on the other column => It prevents the resize to work correctly  :open_mouth: 
Maybe you can remove this class and add the `offset-lg-1` class on this one while you are at it ? :slightly_smiling_face:  => it will give the same visual but it will be easier to resize afterwards :+1: 
Note that I did not check on `xl` screen size, maybe it is why it was needed ? :thinking: 
![image](https://github.com/user-attachments/assets/13665a72-22c1-4157-8f13-1cdc3e2de9c5)

Reply:
Indeed, edition is broken. I adapted it to be the same layout than when form is visible. I think it makes sense :)